src/resonate/promise.py

```diff
@@ -64,9 +64,14 @@ def is_blocked(self) -> bool:
         return self._blocked_on != 0
 
     def is_blocked_on_remote(self) -> bool:
```

Comment:
Let's rename this to `is_blocked_only_on_remote`. Since it's more telling of what we are actually checking. 

What do you think?

Reply:
I think is probably the same. Given that we have is_blocked, which implies it is blocked on remote or local, and then we have blocked_on_remote which means blocked just on remote. Not a big fan a very long names, If you feel it adds clarity I can change it though.

Reply:
Ok. Let's leave it as it is